openvdb_ax/openvdb_ax/test/main.cc

```diff
@@ -37,218 +15,6 @@ int sGenerateAX = false;
-"    -t test   specific suite or test to run, e.g., \"-t TestGrid\"\n" <<
-"              or \"-t TestGrid::testGetGrid\" (default: run all tests)\n" <<
-"    -v        verbose output\n" <<
-"    -g        As well as testing, auto-generate any integration tests\n";
-#ifdef OPENVDB_USE_LOG4CPLUS
-    ostrm <<
-"\n" <<
-"    -error    log fatal and non-fatal errors (default: log only fatal errors)\n" <<
-"    -warn     log warnings and errors\n" <<
-"    -info     log info messages, warnings and errors\n" <<
-"    -debug    log debugging messages, info messages, warnings and errors\n";
-#endif
-}
-
-
-void
-getTestNames(StringVec& nameVec, const CppUnit::Test* test)
-{
-    if (test) {
-        const int numChildren = test->getChildTestCount();
-        if (numChildren == 0) {
-            nameVec.push_back(test->getName());
-        } else {
-            for (int i = 0; i < test->getChildTestCount(); ++i) {
-                getTestNames(nameVec, test->getChildTestAt(i));
-            }
-        }
-    }
-}
-
-
-/// Listener that prints the name, elapsed time, and error status of each test
-class TimedTestProgressListener: public CppUnit::TestListener
-{
-public:
-    void startTest(CppUnit::Test* test) override
-    {
-        mFailed = false;
-        std::cout << test->getName() << std::flush;
-        mTimer.start();
-    }
-
-    void addFailure(const CppUnit::TestFailure& failure) override
-    {
-        std::cout << " : " << (failure.isError() ? "error" : "assertion");
-        mFailed  = true;
-    }
-
-    void endTest(CppUnit::Test*) override
-    {
-        if (!mFailed) {
-            // Print elapsed time only for successful tests.
-            const double msec = std::round(mTimer.milliseconds());
-            if (msec > 1.0) {
-                openvdb::util::printTime(std::cout, msec, " : OK (", ")",
-                    /*width=*/0, /*precision=*/(msec > 1000.0 ? 1 : 0), /*verbose=*/0);
-            } else {
-                std::cout << " : OK (<1ms)";
-            }
-        }
-        std::cout << std::endl;
-    }
-
-private:
-    openvdb::util::CpuTimer mTimer;
-    bool mFailed = false;
-};
-
-
-int
-run(int argc, char* argv[])
-{
-    const char* progName = argv[0];
-    if (const char* ptr = ::strrchr(progName, '/')) progName = ptr + 1;
-
-    bool shuffle = false, verbose = false;
-    StringVec tests;
-    for (int i = 1; i < argc; ++i) {
-        const std::string arg = argv[i];
-        if (arg == "-l") {
-            StringVec allTests;
-            const CppUnit::Test* tests = CppUnit::TestFactoryRegistry::getRegistry().makeTest();
-            getTestNames(allTests, tests);
-            delete tests;
-            for (const auto& name: allTests) { std::cout << name << "\n"; }
-            return EXIT_SUCCESS;
-        } else if (arg == "-shuffle") {
-            shuffle = true;
-        } else if (arg == "-v") {
-            verbose = true;
-        } else if (arg == "-g") {
-            sGenerateAX = true;
```

Comment:
This used to be callable from the test binary - in practice we don't use it very much, so in the rare cases where the tests need to be re-generated, I think it's fine for this to need a re-compile. So just commenting for future visibility, nothing else to do